django/db/backends/mysql/features.py

```diff
@@ -86,6 +86,11 @@ def supports_over_clause(self):
     def has_select_for_update_skip_locked(self):
         return self.connection.mysql_version >= (8, 0, 1)
 
+    @cached_property
+    def requires_using_regexp_like_function(self):
```

Comment:
not sure this needs a 'feature', you can just do the check on the version in `regex_lookup()`